.env

```diff
@@ -0,0 +1,9 @@
+PORT = 3000;
```

Comment:
@UwaweTera don't push this file on GitHub. These are your secret keys, you can't expose them to the public.